doc/source/openfisca_serve.rst

```diff
@@ -42,14 +48,16 @@ Serving reforms
 
 Using a configuration file
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
+You can setup `openfisca serve` using a configuration file. Be careful as parameters with a '-' in their name change to an '_' when used from the config file.
 
 **config.py:**
 
 .. code-block:: py
 
   port = 4000
   workers = 4
-  country-package = 'openfisca_france'
+  bind = 0.0.0.0:4000
```

Comment:
Update to `bind = '0.0.0.0:' + str(port)` as bind takes a string value and `port` variable is defined above?

> `config.py` is an extension of [gunicorn configuration file](http://docs.gunicorn.org/en/stable/configure.html#configuration-file). It manages openfisca specific configuration like `port`. `port` value will also update `bind` value (openfisca configuration has higher priority here than gunicorn's configuration).